### DISCUSSION TITLE
model-viewer is not working as intended,on custom web built in service like Wix

### DISCUSSION THREAD

**Discussion by kotran88** [google/model-viewer#3520] [Q&A]
Hello :)

Model-viewer works like charm on my domain! 
and I made homepage with service like Wix. 
it based on HTML, JS so I think I can show model with model-viewer and also AR . 
and it works on model-preview, I can rotate them as intended. 
but the problem is on AR mode by pressing button . 

it strangely redirect me to play store of Google Play AR Service which I already installed. 

what may be the problem? 

![](https://im3.ezgif.com/tmp/ezgif-3-7c257d4e07.gif)

**Comment by elalish** [comment]
I don't know; maybe it's out of date? Can you try some other WebXR experiences and verify if they work?
